HW3/src/main/kotlin/part2.kt

```diff
@@ -5,5 +5,5 @@ fun sString(){
    var phrase = "The quick brown fox jumps over the lazy dog"
 
     println(phrase.substring(0,15))
```

Comment:
This line does not showing `brown fox jumps `

Reply:
It's work with me !!

Reply:
oh, ok sorry